components/Drawer.tsx

```diff
@@ -23,6 +23,7 @@ interface Props {
 }
 
 const URL_APPLY = "/apply";
+const URL_MENTORSHIP_HOME = "/";
 const URL_HOME = "https://advisory.sg/";
```

Comment:
Just a minor remark - could "/" be `URL_HOME` and "https://advisory.sg" be `URL_MAIN` instead?